vmss-prototype/vmss-prototype

```diff
@@ -1026,7 +1027,7 @@ def vmss_prototype_update(sub_args):
     for extension in extensions:
         # Now that we have a disk image that already has all of our
         # provisioning bits on it (prototype) we can remove all extensions
-        # But...  We want to keep the billing one for telemetry reasons
+        # But...  We want to keep the AKS Engine-identifying extension for telemetry reasons
```

Comment:
At some point we should talk about how aks-engine telemetry is collected.

Reply:
That would allow us to remove this specific exception for extensions.
And maybe add a potential set of extensions that you don't want removed (optional)

However, the core goal is to remove all of those extensions/costs on a per-vm scale scenario.